### PR TITLE
[skip-ci] Format the tables to look similar than the ones in Doxygen

### PR DESCRIPTION
**Pull request by bellenot**

### CODE REVIEW

File: io/doc/TFile/header.md [hunk 1]
```diff
@@ -1,48 +1,63 @@
...
-      37->40 SeekInfo    = Byte offset of StreamerInfo record         TFile::fSeekInfo
-      41->44 NbytesInfo  = Number of bytes in StreamerInfo record     TFile::fNbytesInfo
-      45->63             = Unused??
-</pre></div>
\ No newline at end of file
+<div class="byteTable">
+
+| Byte Range       | Record Name    | Description                             | |
+|------------------|----------------|-----------------------------------------|-|
+| 0...3            | "root"      | Identifies this file as a ROOT file        | |
```
Comment (pcanal) [inline io/doc/TFile/header.md:47]:
```suggestion
|------------------|----------------|-----------------------------------------|-|
| 0...3            | "root"         | Identifies this file as a ROOT file     | |
```
(or whatever white space tweaks is needed to have a decent looking table even when reading the raw ascii text (with a fixed size font)
thanks.

Comment (bellenot) [reply io/doc/TFile/header.md]: Thanks Philippe, Should be better now

**Comment by bellenot** [comment]
Wait, it looks like I made a mistake...

**Review by eguiraud — APPROVED** [review]
LGTM!

**Review by pcanal — APPROVED** [review]

File: io/doc/TFile/header.md [hunk 2]
```diff
@@ -1,48 +1,61 @@
 \page header File header format
 
+<style>
```
Comment (couet) [inline io/doc/TFile/header.md:3]: You do not need this style definition

**Pull request merged**